README.md

```diff
@@ -1,35 +1,78 @@
-# Gluco Check
+# Welcome to Gluco Check
 
-formerly Nightscout Assistant / Nightscout Status
+#### formerly [Nightscout Status](https://github.com/nielsmaerten/nightscout-assistant) 
+[![CodeFactor Grade](https://img.shields.io/codefactor/grade/github/nielsmaerten/gluco-check/main?style=flat-square)](https://www.codefactor.io/repository/github/nielsmaerten/gluco-check/)
+[![GitHub Workflow Status](https://img.shields.io/github/workflow/status/nielsmaerten/gluco-check/Build,%20Test,%20Lint,%20Deploy?style=flat-square)](https://github.com/nielsmaerten/gluco-check/actions?query=workflow%3A%22Build%2C+Test%2C+Lint%2C+Deploy%22)
+[![Codecov](https://img.shields.io/codecov/c/github/nielsmaerten/gluco-check?style=flat-square)](https://codecov.io/gh/nielsmaerten/gluco-check)
 
-## Hey Google, talk to Gluco Check
+## Getting started
 
-> "Blood sugar is 102 and rising slowly as of 4 minutes ago."
+<!-- TODO: Update URL to Production version -->
+1. [Link your Nightscout site](https://nightly.diabase.app)
+2. [Ask for beta access](https://gitter.im/nielsmaerten)
+3. Say: **"Hey Google, talk to Gluco Check"**  
 
-## Bèta testing is now open
+## Things you can ask Gluco Check
+
+### Blood sugar
+
+Hey Google, ask Gluco Check my blood sugar
+
+### Insulin on Board / Carbs on Board
+
+Hey Google, ask Gluco Check how many carbs I have  
+Hey Google, ask Gluco Check my IOB  
+
+### Sensor age / Infusion set age
+
+Hey Google, ask Gluco Check when I last changed my sensor  
+Hey Google, ask Gluco Check my cannula's age
+
+### Pump battery level / Pump reservoir level
+
+Hey Google, ask Gluco Check my pump's battery  
+Hey Google, ask Gluco Check how many units I have left
```

Comment:
Hey Google, ask Gluco Check how much insulin I have left
Hey Google, ask Gluco Check how much insulin is left
Hey Google, ask Gluco Check how much insulin is left in my pump

Reply:
Same problem here.
"Hey Google, ask Gluco Check how much insulin I have left" could be interpreted 2 ways:
- How much insulin do I have left on board (=IOB)
- How much insulin is left in my pump (=reservoir)